packages/core/src/components/tabs/tabs.tsx

```diff
@@ -250,7 +259,6 @@ export class Tabs {
         'selected',
         index === this.selected ? 'true' : 'false'
       );
-      element.setAttribute('placement', this.placement);
```

Comment:
Can we move the whole forEach block into `componentWillLoad`?

Reply:
I tried moving everything outside of componentDidRender